sharedbuffers/mapped_struct.py

```diff
@@ -977,10 +977,9 @@ def __iter__(self):
     @cython.locals(l=cython.longlong)
     def __reversed__(self):
         l = len(self)
-        if l == 0:
-            return reversed(self._make_empty())
-        for i in xrange(l - 1, -1, -1):
-            yield self[i]
+        if l > 0:
+            for i in xrange(l - 1, -1, -1):
+                yield self[i]
```

Comment:
If it is `empty` it is going to retrun None? Shouldn' it return a empty value?

Reply:
Since this is a generator, the implicit returning of None is meant to stop iteration.